index.html

```diff
@@ -37,31 +44,40 @@ <h1 id="title"> freecode survay </h1>
 
 <p> what describes your choice option </p>
     <div class= "first"> 
+
       <select id= "dropdown" size="4"> 
+
 <option> first choice </option>
 <option> second choice </option>
 <option> third choice </option>
 <option> forth choice </option>
-<option> fifth choice </option>
-<option> sixth choice </option>
+
+<!-- <option> fifth choice </option> -->
+<!-- <option> sixth choice </option> -->
```

Comment:
kindly ensure you don't have unused code as comments. Unused code makes your work not clean